### ISSUE TITLE
Unsynchronized A/V after progressing to the next lesson with higher playback rates

### ISSUE THREAD

**Issue by dachev** [FrontendMasters/ios-issues#65]
## Issue Title
Audio and video become unsynchronized after progressing to the next lesson if the playback rate is > 1

### Steps to Reproduce
The most import part.
```
1. Play a lesson at 2x
2. Wait for lesson the end and start playing the next one
```
### Actual and Expected Result
```
Expected:
Playback as usual

Actual:
Audio and video may go unsynchronized
```

ref:
  * https://app.intercom.com/a/inbox/w1ukcwje/inbox/conversation/12149500163211
  * https://app.intercom.com/a/inbox/w1ukcwje/inbox/conversation/12149500191076
  * https://app.intercom.com/a/inbox/w1ukcwje/inbox/conversation/12149500190952

**Comment by dachev** [comment]
@kenearley this should be fixed in the latest staging build: 1.30.0 (524)

**Comment by kenearley** [comment]
This is working well for me :shipit:

**Comment by dachev** [comment]
This should be fixed in 1.30.0
